src/mrt/runners.rs

```diff
@@ -0,0 +1,38 @@
+use serde::{Serialize, Deserialize};
+
+use super::package::Package;
+
+pub struct PackageScriptRunContext<'a> {
+    pub package: &'a Package,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum PackageScriptRunResultType {
+    Success,
+    Error(String),
+    Noop
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PackageScriptRunResult {
+    pub result_type: PackageScriptRunResultType,
+    pub stdout: String,
```

Comment:
You probably want exit code here.

Reply:
Not yet sure, idea was to use PackageScriptRunResultType so that runner will actually decide on it's own if it was correct. I added exit code, just in case :)